doc/heps/hep2.md

```diff
@@ -0,0 +1,370 @@
+# HEP 2: Release and deprecation policies
+
+<table>
+<tr><td> Identifier </td><td> HEP 2 </td>
+<tr><td> Title </td><td> Release and deprecation policies </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Maxime Liquet </td></tr>
+<tr><td> Created </td><td> TBD </td></tr>
+<tr><td> Updated </td><td> - </td></tr>
+<tr><td> Discussion </td><td> https://github.com/holoviz/holoviz/pull/388 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Summary
+
+This HEP describes the release and deprecation policies adopted by the HoloViz Projects. While two HEPs could have be written to describe these policies separately, it is deemed that they are enough intertwined to be addressed together (e.g. to answer questions such as in which kind of final release can a feature be deprecated/removed?).
```

Comment:
I think the 'Summary' and 'Motivation' are a bit too overlapping. I'd suggest simplifying the Summary such as follows:

"This HEP outlines unified release and deprecation policies for the HoloViz Projects. It  addresses key questions about the lifecycle of features, including deprecation and removal timelines. The HEP formalizes existing release procedures and introduces well-defined deprecation guidelines to streamline long-term maintenance and decision-making processes, aiming for consistency across projects and an improved user experience."

Reply:
Thanks for the suggestion! Updated in https://github.com/holoviz/holoviz/pull/388/commits/7d42f6f99afc0dc18f3b42724c0a457f8e8d327b.